hal/audio_extn/cirrus_sony.c

```diff
@@ -592,13 +594,46 @@ static int cirrus_play_silence(int seconds) {
     struct audio_device *adev = handle.adev_handle;
     struct mixer_ctl *ctl_config = NULL;
     struct pcm_config rx_tmp = { 0 };
+    struct audio_usecase *uc_info_rx;
+    #define CIRRUS_PLAYBACK_MIXER "SEN_MI2S_RX Audio Mixer MultiMedia1"
```

Comment:
Sorry, your development notes are not allowed... :P :P :P :P :P

Reply:
Lmao that's remaining from a readback from this thing, to show that is actually getting activated. Gross!

Reply:
Yeah, I also do that rather frequently. I know :D